src/modal/modal-container.component.ts

```diff
@@ -88,8 +88,12 @@ export class ModalContainerComponent implements OnInit, OnDestroy {
     this.hide();
   }
 
-  @HostListener('window:keydown.esc')
-  onEsc(): void {
+  @HostListener('keydown.esc', ['$event'])
```

Comment:
if focus will be outside of modal 
what is possible without focus trap in place
pressing esc will not close modal